.travis/deploy_npm

```diff
@@ -36,22 +36,56 @@ fi
 exe node -v
 exe npm  -v
 
+echo ">>> Installing dependencies..."
 exe npm install
-exe npm install --no-save @qooxdoo/compiler
-
-exe npx qx --version
-exe npx qx pkg install -v 
-#compile everything
-exe npx qx compile --config-file=compileServer.json --target=build 
-#exe npx qx compile --config-file=compile.json       --target=build 
-#remove unnecessary stuff
-#exe find ./apps/transpiled -type f -name '*.map' -delete
-#exe find ./apps/transpiled -type f -name '*.js'  -delete
-#copy source
+
+echo ">>> Installing github.com/qooxdoo/qooxdoo-compiler"
+exe rm -rf ./qooxdoo-compiler
+exe git clone https://github.com/qooxdoo/qooxdoo-compiler.git --depth=1 --single-branch ./qooxdoo-compiler
+exe cd ./qooxdoo-compiler
+exe npm install
+QX=`pwd`/qx
+exe cd ..
+exe $QX --version
+
+echo ">>> Compiling & deploying server library with the compiler master using current qx_server library from NPM..."
+exe $QX compile --config-file=compile-server.json --target=build
+exe mkdir -p ./lib
+exe rm -rf ./lib/*
+exe cp -R compiled/build/qx_server ./lib
+
+echo ">>> Running Compiler Tests"
```

Comment:
This tests are runing with the npm Version library version. Should'nt the new library not be copied before the tests?

Reply:
True, the tests need to be moved down to after line 66, I think.